CHANGELOG.md

```diff
@@ -2,6 +2,46 @@
 
 ## Unreleased
 
+### AWS X-Ray SDK support
+
+- Update semconv dependency version
+  ([#1585](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1585))
+
+### Baggage processor
+
+- [baggage-processor] Add BaggageLogRecordProcessor
+  ([#1576](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1576))
+
+### Disk buffering
+
+- Deserialization validation
+  ([#1571](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1571))
+
+### JMX metrics
+
+- Align HBase metric units to semconv
+  ([#1538](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1538))
+- Align Cassandra metric units to semconv
+  ([#1591](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1591))
+- Align Tomcat metric units to semconv
+  ([#1589](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1589))
+- Align JVM units to semconv
+  ([#1593](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1593))
+
```

Comment:
There is one more: https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1553

Reply:
oh, it looks like these were included in the last release, but I missed adding them to the change log / release notes

I'll send a separate PR to update the change log for 1.41.0 and I'll manually update the release notes